pkg/polaris/patcher.go

```diff
@@ -0,0 +1,192 @@
+	} else {
+		content = strings.ReplaceAll(content, "${POSTGRES_TYPE}", "external")
+	}
+
+	// Reporting
+	if polaris.EnableReporting {
+		content = strings.ReplaceAll(content, "${REPORTING_URL}", fmt.Sprintf("https://%s/reporting", polaris.EnvironmentDNS))
+	} else {
+		content = strings.ReplaceAll(content, "${REPORTING_URL}", "")
+	}
+
+	// Org job
+	content = strings.ReplaceAll(content, "${ORG_DESCRIPTION}", polaris.OrganizationDetails.OrganizationProvisionOrganizationDescription)
+	content = strings.ReplaceAll(content, "${ORG_NAME}", polaris.OrganizationDetails.OrganizationProvisionOrganizationName)
+	content = strings.ReplaceAll(content, "${ADMIN_NAME}", polaris.OrganizationDetails.OrganizationProvisionAdminName)
+	content = strings.ReplaceAll(content, "${ADMIN_USERNAME}", polaris.OrganizationDetails.OrganizationProvisionAdminUsername)
+	content = strings.ReplaceAll(content, "${ADMIN_EMAIL}", polaris.OrganizationDetails.OrganizationProvisionAdminEmail)
+	content = strings.ReplaceAll(content, "${SEAT_COUNT}", polaris.OrganizationDetails.OrganizationProvisionLicenseSeatCount)
+	content = strings.ReplaceAll(content, "${TYPE}", polaris.OrganizationDetails.OrganizationProvisionLicenseType)
+	content = strings.ReplaceAll(content, "${RESULTS_START_DATE}", polaris.OrganizationDetails.OrganizationProvisionResultsStartDate)
+	content = strings.ReplaceAll(content, "${RESULTS_END_DATE}", polaris.OrganizationDetails.OrganizationProvisionResultsEndDate)
+	content = strings.ReplaceAll(content, "${RETENTION_START_DATE}", polaris.OrganizationDetails.OrganizationProvisionRetentionStartDate)
+	content = strings.ReplaceAll(content, "${RETENTION_END_DATE}", polaris.OrganizationDetails.OrganizationProvisionRetentionEndDate)
+
+	// TODO https://github.com/blackducksoftware/synopsys-operator/issues/943
+	if len(polaris.Repository) != 0 {
+		content = strings.ReplaceAll(content, "gcr.io/snps-swip-staging", polaris.Repository)
+	}
+
+	mapOfUniqueIDToBaseRuntimeObject := ConvertYamlFileToRuntimeObjects(content)
+	mapOfUniqueIDToBaseRuntimeObject = removeTestManifests(mapOfUniqueIDToBaseRuntimeObject)
+
+	patcher := Patcher{
+		polaris:                          polaris,
+		mapOfUniqueIDToBaseRuntimeObject: mapOfUniqueIDToBaseRuntimeObject,
+	}
+
+	return patcher.patch()
+}
+
+type Patcher struct {
+	polaris                          Polaris
+	mapOfUniqueIDToBaseRuntimeObject map[string]runtime.Object
+}
+
+func (p *Patcher) patch() (map[string]runtime.Object, error) {
+	patches := []func() error{
+		p.patchNamespace,
+		p.patchVersionLabel,
+	}
+	for _, f := range patches {
+		err := f()
+		if err != nil {
+			return nil, err
+		}
+	}
+	return p.mapOfUniqueIDToBaseRuntimeObject, nil
+}
+
+func (p *Patcher) patchNamespace() error {
+	accessor := meta.NewAccessor()
+	for _, runtimeObject := range p.mapOfUniqueIDToBaseRuntimeObject {
+		accessor.SetNamespace(runtimeObject, p.polaris.Namespace)
+	}
+	return nil
+}
```

Comment:
why do we need this now since we have placeholder?

Reply:
It has been removed